packages/react/src/backlink/Backlink.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta } from '@storybook/react';
+
+import { Backlink } from '.';
+
+const meta: Meta<typeof Backlink> = {
+  title: 'Backlink',
+  component: Backlink,
+};
+
+export default meta;
+
+export const Default = () => {
+  return <Backlink href="#">Tillbake</Backlink>;
+};
+
+export const WithCustomIcon = () => {
+  return (
+    <Backlink href="#" iconStyle="text-green">
+      Tillbake
+    </Backlink>
+  );
+};
+
+export const NoUnderline = () => {
+  return (
+    <Backlink href="#" className="no-underline">
```

Comment:
Burde dette heller innkapsles som en variant? I hvert fall hvis dette er noe som brukes i Figma? Hva tenker dere?

Enten som `variant` eller `textDecoration` eller noe annet kanskje?

Reply:
Var inne på noe av det samme. Siden det er underline som er default styling i RAC: hva med en prop for `withUnderline?: boolean` som default er `true`?

Reply:
Etter diskusjon i standup ble vi enige om å release dette sånn som det er nå (settes mha. `className`). Konklusjonen var at vi heller kan utvide med ekstra props senere uten at vi introduserer breaking changes.

Reply:
Og etter en diskusjon med Kristian kom det frem at underline som default kun skal være på hover. Og da må vi nesten styre dette med en prop.

Reply:
Det ble også bestemt at dette skal være samme som i ny meny (der vi bruker custom underline). Så det er enda en grunn til at vi må bruke en prop.